server/views/pages/bookAVisit/dateAndTime.njk

```diff
@@ -21,7 +21,7 @@
     {%- set tableText %}
       {% if (doubleBooked and not checked) -%}
         Prisoner has a visit
-      {%- elseif slot.availableTables === 0 -%}
+      {%- elseif slot.availableTables <= 0 -%}
```

Comment:
Handle availability potentially being temporarily below zero while a visit is being updated.